src/components/FolderPage/CardListItem/CardListItem.js

```diff
@@ -3,31 +3,110 @@ import { getTimeAgo, formatDate } from "util/time";
 import noImg from "assets/noImg.png";
 import starImg from "assets/Star 1.svg";
 import kebabImg from "assets/kebab.svg";
+import { useEffect, useRef, useState } from "react";
+
+function CardListItem({ link, setModal, setLink }) {
+  const [btnclicked, setBtnClicked] = useState(false);
+  const searchRef = useRef(null);
+
+  useEffect(() => {
+    function handleFocus(e) {
+      if (searchRef.current && !searchRef.current.contains(e.target)) {
+        setBtnClicked(false);
+      }
+    }
+
+    document.addEventListener("mouseup", handleFocus);
+    return () => {
+      document.removeEventListener("mouseup", handleFocus);
+    };
+  }, [searchRef]);
+
+  const handleStarBtnClick = (event) => {
```

Comment:
(마이너)
네이밍을 UI형태를 표현하기보다 행위에 초점을 맞추면 유추하기 더 쉬울것 같슴니다!
```suggestion
  const handleAddBookmark = (event) => {
```